rex_xai/resp_maps.py

```diff
@@ -93,7 +97,12 @@ def update_maps(
                 raise ReXMapError("the provided mutant has no known classification")
             # check if k has been seen before and has a map. If k is new, make a new map
             if k not in self.maps:
-                self.new_map(k, data.model_height, data.model_width)
+                if data.model_depth is not None:
+                    self.new_map(
+                        k, data.model_height, data.model_width, data.model_depth
+                    )
+                else:
+                    self.new_map(k, data.model_height, data.model_width)
```

Comment:
Minor: the `new_map`method can handle depth=None already, so I don't think you strictly need this additional if-else statement